JobConfig/cosmic/ExtractedNoFieldCRY.fcl

```diff
@@ -9,3 +9,4 @@ physics.producers.generate : {
 outputs.PrimaryOutput.fileName : "dts.owner.CosmicCRYExtracted.version.sequencer.art"
 services.GeometryService.inputFile: "Production/JobConfig/cosmic/geom_cosmic_extracted.txt"
 services.GeometryService.bFieldFile: "Offline/Mu2eG4/geom/bfgeom_no_field.txt"
+physics.filters.PrimaryFilter.CrvSteps: [ "CrvSteps" ]
```

Comment:
Can you address this as well?  in primary.prolog.fcl, line 54:
      MinimumCrvStepEnergy : 1.0e-6 # get a real value from CRV experts FIXME!  
Also please override the code default value for this:
fhicl::Atom<unsigned> minNCrvSteps { Name("MinimumCrvSteps"), Comment("Minimum number of good CRV steps"), 3 };